### PR TITLE
BaSP-MR-17: Set up Week 10

### PR DESCRIPTION
**Pull request by francoax**
Development: 

- Production dependencies and devDependencies installed.
- New folders added
- New files added
- Index and package.json updated

**Note:**
Every file test has a line "test.skip" that need to be deleted

### CODE REVIEW

**Review by Eliezer-Alberto — APPROVED** [review]
Set up Ok.

**Review by CavalloFede — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-app.use(cors());
-app.use(express.json());
-mongoose.connect(DB_URL)
-  .then(() => console.log('CONNECTED DB'))
-  .catch((error) => console.log('Error: ', error));
-app.use('/api', router);
-
-app.listen(port, () => {
-  console.log(`Example app listening on port ${port}`);
+app.get('/', (req, res) => {
```
[the remark this answers is not in the document]
Comment (francoax) [reply src/index.js:13]: The new index.js that were sent to me, it has it

[the remark this answers is not in the document]
Comment (Micaela-Rossi) [reply src/index.js:13]: Okay

**Review by IgnacioGalceran — APPROVED** [review]
Everything is allright for me

**Review by nailaghigo — APPROVED** [review]
💯

**Pull request merged**